src/hapi/pipelines/utilities/org.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from hapi.pipelines.database.db_org import DBOrg
+
+logger = logging.getLogger(__name__)
+
+
+class Org:
+    def __init__(self, session: Session):
+        self._session = session
+        self.data = {}
+
+    def populate_single(
+        self,
+        acronym,
+        orgname,
+        orgtype,
+        reference_period_start,
+        reference_period_end,
+    ):
+        logger.info(f"Adding org {orgname}")
+        # TODO: add HDX link for orgs. What if there is no hdx link?
```

Comment:
Oops, HDX link should not be in the orgs table! You can pull it out of the schema yourself, or once you merge the main branch in we'll need to make sure it's fixed in Simon's new hapi-schema library

Reply:
Just removed it.